extensions/email/src/main/java/org/mule/extension/email/internal/operations/ReplyOperation.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.email.internal.operations;
+
+import static org.mule.extension.email.internal.util.EmailUtils.getAttributesFromMessage;
+import org.mule.extension.email.api.EmailContent;
+import org.mule.extension.email.internal.EmailAttributes;
+import org.mule.extension.email.internal.builder.MessageBuilder;
+import org.mule.extension.email.internal.exception.EmailSenderException;
+import org.mule.runtime.api.message.MuleMessage;
+
+import java.util.Calendar;
+import java.util.List;
+
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.Transport;
+
+/**
+ * Represents the reply operation.
+ *
+ * @since 4.0
+ */
+public final class ReplyOperation
+{
+
+    public static final String REPLY_ERROR = "No email attributes were found in the incoming message.";
```

Comment:
shouldn't this error be centralized on the EmailUtils class you use to extract the attributes out of the MM?